content-repo/extra-docs/releases/20.5.2.md

```diff
@@ -0,0 +1,271 @@
+## Cortex XSOAR Release Notes for version 20.5.2 (54345)
+##### Published on 26 May 2020
+
+### Breaking Changes
+- __TrueSTAR__  
+Changed several arguments and outputs in **trustar-get-phishing-submissions** and **trustar-get-phishing-indicators** commands. See the **Integrations** section for full details.
```

Comment:
lets make the reference to the integrations clickable by adding a link to the integration section